src/nvim/window.c

```diff
@@ -6145,6 +6167,110 @@ void set_fraction(win_T *wp)
   }
 }
 
+/*
+ * Handle scroll position for 'nosplitscroll', replaces
+ * scroll_to_fraction() call from win_new_height().
+ * Instead we iterate over all windows in a tabpage.
+ */
+static void spsc_correct_scroll(win_T *next_curwin, int flags)
```

Comment:
- `fix_cursor` is an existing, less verbose precedent.
- "spsc" is pretty obscure. "win_" is a common prefix that fits here.

```suggestion
static void win_fix_scroll(win_T *next_curwin, int flags)
```